src/components/color-picker/color-picker.component.html

```diff
@@ -43,7 +43,7 @@
             <button type="button"
                 attr.aria-label="Switch input mode to {{inputMode === 'hex' ? 'RGBA' : 'hex'}}" i18n-aria-label
                 class="btn btn-link btn-icon button-secondary ux-color-picker-input-toggle"
-                (click)="toggleColorEntryType(); $event.stopPropagation()">
+                (click)="toggleColorEntryType(); $event.preventDefault()">
```

Comment:
Have you checked if this change is still required with the NgModel changes below?

Reply:
Doesn't seem to be needed, shall I revert back to $event.stopPropagation()

Reply:
Yes, thanks, just in case it has some side effects that we didn't expect.